packages/vscode-extension/src/test/suite/extension.test.ts

```diff
@@ -0,0 +1,15 @@
+import * as assert from 'assert';
+
+// You can import and use all API from the 'vscode' module
+// as well as import your extension to test it
+import * as vscode from 'vscode';
+// import * as myExtension from '../extension';
```

Comment:
Delete comment

Reply:
The comment was auto-generated in the scafolding of the vscode-extension.